profiles/views_test.py

```diff
@@ -397,3 +401,101 @@ def test_upload_image(self):
         assert resp.status_code == 200, resp.content.decode('utf-8')
         assert profile.education.count() == 1
         assert profile.work_history.count() == 1
+
+    @ddt.data(
+        *itertools.product([True, False], [True, False])
+    )
+    @ddt.unpack
+    def test_no_thumbnail_change_if_image_upload(self, image_already_exists, thumb_already_exists):
+        """
+        A patch without an image upload should not touch the image or the thumbnail
+        """
+        with mute_signals(post_save):
+            profile = ProfileFactory.create(user=self.user1, filled_out=False, agreed_to_terms_of_service=False)
+            if image_already_exists is False:
+                profile.image = None
+            if thumb_already_exists is False:
+                profile.image_small = None
+            profile.save()
+        self.client.force_login(self.user1)
+
+        patch_data = ProfileSerializer(profile).data
+        del patch_data['image']
+        del patch_data['image_small']
+
+        resp = self.client.patch(self.url1, content_type="application/json", data=json.dumps(patch_data))
+        assert resp.status_code == 200
+
+        profile.refresh_from_db()
+        assert bool(profile.image) == image_already_exists
+        assert bool(profile.image_small) == thumb_already_exists
+
+    @ddt.data(
+        [True, False],
+        [True, False]
```

Comment:
You're right. This was meant to be every combination of true, false for the two arguments. I fixed this for the other test but I overlooked this one